### PR TITLE
Replace Twitter social icon with Bsky

### PR DESCRIPTION
**Pull request by snim2**
To match the current dxw.com.

## How to test

Run the site locally and check the footer links to the dxw bsky account.

### Screenshots

Was:

Screenshot 2024-10-17 at 13 04 34

Is now:

Screenshot 2024-10-17 at 16 05 49

### CODE REVIEW

File: wp-content/themes/dxw-security-2017/templates/partials/global-footer.php [hunk 1]
```diff
@@ -57,11 +57,11 @@
...
...="#eeeeee" d="M 70.796875 50.65625 C 70.226562 50.585938 69.636719 50.519531 69.0625 50.433594 C 69.652344 50.5 70.226562 50.585938 70.796875 50.65625 Z M 50 39.035156 C 45.46875 30.324219 33.140625 14.082031 21.683594 6.070312 C 10.695312 -1.613281 6.511719 -0.289062 3.75 0.949219 C 0.574219 2.375 0 7.207031 0 10.042969 C 0 12.878906 1.578125 33.347656 2.605469 36.769531 C 5.988281 48.058594 18.074219 51.875 29.203125 50.636719 C 29.773438 50.550781 30.347656 50.484375 30.9375 50.398438 C 30.363281 50.484375 29.792969 50.570312 29.203125 50.636719 C 12.898438 53.042969 -1.578125 58.921875 17.414062 79.839844 C 38.296875 101.253906 46.023438 75.25 50 62.066406 C 53.976562 75.25 58.542969 100.308594 82.222656 79.839844 C 100 62.066406 87.101562 53.027344 70.796875 50.636719 C 70.226562 50.570312 69.636719 50.5 69.0625 50.414062 C 69.652344 50.484375 70.226562 50.570312 70.796875 50.636719 C 81.925781 51.859375 93.992188 48.042969 97.394531 36.769531 C 98.421875 33.347656 100 12.894531 100 10.042969 C 100 7.1875 99.425781 2.359375 96.25 0.949219 C 93.507812 -0.269531 89.304688 -1.613281 78.332031 6.070312 C 66.859375 14.082031 54.53125 30.324219 50 39.035156 Z M 50 39.035156"/></svg>
```
Comment (snim2) [inline wp-content/themes/dxw-security-2017/templates/partials/global-footer.php:62]: There are possibly better ways to do this than messing with the viewport?

File: wp-content/themes/dxw-security-2017/templates/partials/global-footer.php [hunk 2]
```diff
@@ -57,11 +57,11 @@
...
```
Comment (snim2) [inline wp-content/themes/dxw-security-2017/templates/partials/global-footer.php:62]: There are possibly better ways to do this than messing with the viewport?

Comment (sbrody) [reply wp-content/themes/dxw-security-2017/templates/partials/global-footer.php:62]: There's not much you can do with positioning once the svg has been created - it's all set in the co-ordinates. So unless we want to recreate it at the centre of a larger canvas as the others seem to be, I think it's an ok way to go

Comment (sbrody) [reply wp-content/themes/dxw-security-2017/templates/partials/global-footer.php:62]: However, something about it seems to have offset the list item slightly on smaller screens

Comment (sbrody) [reply wp-content/themes/dxw-security-2017/templates/partials/global-footer.php:62]: It gets fixed if you set the ul to be display flex and justify content center

**Comment by snim2** [comment]
Needs squashing before merge

**Review by sbrody — COMMENTED** [review]
The icons look a bit strange stacked up on mobile - I'm not sure if you need the flex display on the list items as you've fixed the svg

**Comment by snim2** [comment]
> The icons look a bit strange stacked up on mobile - I'm not sure if you need the flex display on the list items as you've fixed the svg

So this is how it looks if I revert that and squish the screen size:

Screenshot 2024-10-21 at 15 52 53

**Comment by snim2** [comment]
> > > The icons look a bit strange stacked up on mobile - I'm not sure if you need the flex display on the list items as you've fixed the svg
> > 
> > 
> > So this is how it looks if I revert that and squish the screen size:
> 
> That's only happening for me when I resize below 300px - which is narrower than any modern phone - so I would say that's ok?

Yes, I think it's fine tbh

**Review by sbrody — APPROVED** [review]

**Comment by snim2** [comment]
Rebasing before merge

**Pull request merged**